diff --git a/starters/nextjs-starter-ts/components/grid.tsx b/starters/nextjs-starter-ts/components/grid.tsx
index 420f7b5..34e69cb 100644
--- a/starters/nextjs-starter-ts/components/grid.tsx
+++ b/starters/nextjs-starter-ts/components/grid.tsx
@@ -1,97 +1,115 @@
+import type { ArticleWithoutContent } from "@pantheon-systems/pcc-react-sdk";
 import Link from "next/link";
-import { Tags } from "./tags";
+import { cn } from "../lib/utils";
+import { Button } from "./ui/button";
 
-const GradientPlaceholder = () => (
-  <div className="w-full h-full bg-gradient-to-b from-blue-100 to-blue-500" />
-);
-
-interface Props {
-  href: string;
-  imgSrc: string;
-  altText?: string;
-  tags?: string[];
-  title: string;
-  snippet?: string;
-}
-
-const GridItem = ({ href, imgSrc, altText, tags, title, snippet }: Props) => {
+export function HomepageArticleGrid({
+  articles,
+}: {
+  articles: ArticleWithoutContent[];
+}) {
   return (
-    <div className="flex flex-col h-full overflow-hidden rounded-lg shadow-lg">
-      <Link passHref href={href}>
-        <div className="relative flex-shrink-0 h-40 cursor-pointer hover:border-indigo-500 border-2s not-prose">
-          {imgSrc != null ? (
-            // eslint-disable-next-line @next/next/no-img-element
-            <img
-              src={imgSrc}
-              alt={altText || title}
-              className="object-cover w-full h-full"
-            />
-          ) : (
-            <GradientPlaceholder />
-          )}
-          {snippet != null ? <div>{snippet.substring(0, 120)}</div> : null}
-        </div>
-      </Link>
-      <div className="mx-6 my-4 text-xl font-semibold leading-7 text-gray-900">
-        <Link passHref href={href}>
-          <div className="hover:scale-105">{title} &rarr;</div>
-        </Link>
-        <Tags tags={tags || []} />
-      </div>
+    <div
+      className={cn(
+        "grid grid-cols-1 gap-6 sm:grid-cols-2 lg:w-2/3 2xl:w-full 2xl:grid-cols-[repeat(auto-fit,minmax(300px,438px))] 2xl:justify-center",
+      )}
+    >
+      {articles.map((article, index) => (
+        <ArticleGridCard
+          key={article.id}
+          article={article}
+          isWide={articles.length === 1 || (articles.length > 2 && index === 2)}
+        />
+      ))}
     </div>
   );
-};
+}
 
-const ArticleGridItem = ({ content: article, basePath = "/articles" }) => {
+export function ArticleGrid({
+  articles,
+}: {
+  articles: ArticleWithoutContent[];
+}) {
   return (
-    <GridItem
-      href={`${basePath}/${article.slug || article.id}`}
-      imgSrc={article.metadata?.["Hero Image"]}
-      title={article.title}
-      tags={article.tags}
-      snippet={article.snippet}
-    />
+    <div className={cn("grid grid-cols-1 gap-8 lg:grid-cols-2 xl:grid-cols-3")}>
+      {articles.map((article) => (
+        <ArticleGridCard key={article.id} article={article} />
+      ))}
+    </div>
   );
-};
+}
+
+interface ArticleGridCardProps {
+  article: ArticleWithoutContent;
+  basePath?: string;
+  imageAltText?: string;
+  isWide?: boolean;
+}
+
+export function ArticleGridCard({
+  article,
+  basePath = "/articles",
+  imageAltText,
+  isWide = false,
+}: ArticleGridCardProps) {
+  const targetHref = `${basePath}/${article.slug || article.id}`;
+  const imageSrc = article.metadata?.["Hero Image"] || null;
 
-export const Grid = ({ children }) => {
   return (
     <div
-      className={`mt-12 grid gap-5 max-w-content mx-auto lg:max-w-screen-lg lg:grid-cols-3`}
+      className={cn(
+        "group flex h-full flex-col overflow-clip rounded-xl shadow-lg ring-1 ring-gray-300/50",
+        isWide
+          ? "sm:col-span-2 sm:flex-row 2xl:col-span-1 2xl:flex-col 2xl:only:col-span-2 2xl:only:flex-row"
+          : "",
+      )}
     >
-      {children}
+      <div
+        className={cn(
+          "aspect-video w-full flex-shrink-0 overflow-hidden sm:h-[196px]",
+          isWide
+            ? "sm:h-full sm:max-w-[49%] 2xl:h-[196px] 2xl:max-w-[100%] 2xl:group-only:h-full 2xl:group-only:max-w-[49%]"
+            : "max-w-[100%]",
+        )}
+      >
+        <GridItemCoverImage
+          imageSrc={imageSrc}
+          imageAltText={imageAltText || `Cover image for ${article.title}`}
+        />
+      </div>
+      <div
+        className={cn(
+          "flex flex-grow flex-col justify-between p-8",
+          isWide && "sm:py-24 2xl:py-8 2xl:group-only:py-24",
+        )}
+      >
+        <div>
+          <h1 className="mb-3 text-xl font-semibold leading-7">
+            {article.title}
+          </h1>
+          {article.metadata?.["Description"] && (
+            <p className="line-clamp-3 min-h-[4.5rem] text-gray-600">
+              {article.metadata?.["Description"]?.toString() || ""}
+            </p>
+          )}
+        </div>
+        <Link href={targetHref} className="mt-8">
+          <Button size="large">View</Button>
+        </Link>
+      </div>
     </div>
   );
-};
-
-interface GriddedComponentProps {
-  data: any[] | null;
-  FallbackComponent?: any | null | undefined;
 }
 
-export const withGrid = (Component) => {
-  const GriddedComponents = ({
-    data,
-    FallbackComponent,
-    ...props
-  }: GriddedComponentProps & any) => {
-    return (
-      <>
-        {data ? (
-          <Grid>
-            {data.map((content, i) => {
-              return <Component key={i} content={content} {...props} />;
-            })}
-          </Grid>
-        ) : FallbackComponent ? (
-          FallbackComponent
-        ) : null}
-      </>
-    );
-  };
-
-  return GriddedComponents;
-};
-
-export const PostGrid = withGrid(ArticleGridItem);
-export const PageGrid = PostGrid;
+function GridItemCoverImage({ imageSrc, imageAltText }) {
+  return imageSrc != null ? (
+    // eslint-disable-next-line @next/next/no-img-element
+    <img
+      src={imageSrc}
+      alt={imageAltText}
+      className="h-full w-full object-cover"
+    />
+  ) : (
+    <div className="h-full w-full bg-gradient-to-t from-neutral-800 to-neutral-100" />
+  );
+}
diff --git a/starters/nextjs-starter-ts/components/page-header.tsx b/starters/nextjs-starter-ts/components/page-header.tsx
index 06eec88..2b0408d 100644
--- a/starters/nextjs-starter-ts/components/page-header.tsx
+++ b/starters/nextjs-starter-ts/components/page-header.tsx
@@ -1,7 +1,7 @@
 export default function PageHeader({ title }) {
-	return (
-		<header className="prose text-2xl mx-auto mt-20">
-			<h1 className="text-center mx-auto">{title}</h1>
-		</header>
-	);
+  return (
+    <header className="mb-8">
+      <h1 className="text-5xl font-bold">{title}</h1>
+    </header>
+  );
 }
diff --git a/starters/nextjs-starter-ts/pages/articles/index.tsx b/starters/nextjs-starter-ts/pages/articles/index.tsx
index a0acba3..87d8973 100644
--- a/starters/nextjs-starter-ts/pages/articles/index.tsx
+++ b/starters/nextjs-starter-ts/pages/articles/index.tsx
@@ -1,9 +1,10 @@
 import {
+  ArticleWithoutContent,
   PCCConvenienceFunctions,
   usePagination,
 } from "@pantheon-systems/pcc-react-sdk";
 import { NextSeo } from "next-seo";
-import { PostGrid } from "../../components/grid";
+import { ArticleGrid } from "../../components/grid";
 import Layout from "../../components/layout";
 import PageHeader from "../../components/page-header";
 import Pagination from "../../components/pagination";
@@ -24,25 +25,23 @@ export default function ArticlesListTemplate({ articles, totalCount, cursor }) {
 
   return (
     <Layout>
-      <NextSeo
-        title="Decoupled Next PCC Demo"
-        description="Generated by create-pantheon-decoupled-kit."
-      />
-      <div className="max-w-screen-lg mx-auto">
-        <section>
-          <PageHeader title="Articles" />
-          <PostGrid contentType="posts" data={currentArticles} />
-          <div className="flex mt-4 flex-row justify-center items-center">
-            <Pagination
-              totalCount={totalCount}
-              pageSize={PAGE_SIZE}
-              currentPage={currentPage}
-              onChange={onPageChange}
-              disabled={fetching}
-            />
-          </div>
-        </section>
-      </div>
+      <NextSeo title="Articles" description="Articles" />
+
+      <section className="max-w-screen-3xl mx-auto px-4 pt-16 sm:w-4/5 md:w-3/4 lg:w-4/5 2xl:w-3/4">
+        <PageHeader title="Articles" />
+
+        <ArticleGrid articles={currentArticles as ArticleWithoutContent[]} />
+
+        <div className="mt-4 flex flex-row items-center justify-center">
+          <Pagination
+            totalCount={totalCount}
+            pageSize={PAGE_SIZE}
+            currentPage={currentPage}
+            onChange={onPageChange}
+            disabled={fetching}
+          />
+        </div>
+      </section>
     </Layout>
   );
 }
diff --git a/starters/nextjs-starter-ts/pages/index.tsx b/starters/nextjs-starter-ts/pages/index.tsx
index 2f37265..163df6c 100644
--- a/starters/nextjs-starter-ts/pages/index.tsx
+++ b/starters/nextjs-starter-ts/pages/index.tsx
@@ -1,77 +1,67 @@
-import {
-  PCCConvenienceFunctions,
-  usePagination,
-} from "@pantheon-systems/pcc-react-sdk";
+import { PCCConvenienceFunctions } from "@pantheon-systems/pcc-react-sdk";
 import { NextSeo } from "next-seo";
 import Image from "next/image";
-import { PostGrid } from "../components/grid";
+import { HomepageArticleGrid } from "../components/grid";
 import Layout from "../components/layout";
-import Pagination from "../components/pagination";
-
-const PAGE_SIZE = 20;
-export default function Home({ articles, totalCount, cursor }) {
-  const {
-    data: currentArticles,
-    onPageChange,
-    fetching,
-    currentPage,
-  } = usePagination({
-    cursor,
-    initialArticles: articles,
-    pageSize: PAGE_SIZE,
-  });
-
-  const HomepageHeader = () => (
-    <div className="flex flex-col mx-auto mt-20 prose sm:prose-xl max-w-fit">
-      <h1 className="h-full text-4xl prose text-center">
-        Welcome to{" "}
-        <a
-          className="text-blue-600 no-underline hover:underline"
-          href="https://nextjs.org"
-        >
-          Next.js!
-        </a>
-      </h1>
-    </div>
-  );
+import { Button } from "../components/ui/button";
 
+export default function Home({ articles }) {
   return (
     <Layout>
       <NextSeo
-        title="Decoupled Next PCC Demo"
-        description="Generated by create-pantheon-decoupled-kit."
+        title="Pantheon Content Publisher Next.js Starter"
+        description="A starter kit for building a Next.js site with Pantheon Content Publisher."
       />
-      <HomepageHeader />
-      <section>
-        <PostGrid contentType="posts" data={currentArticles} />
-        <div className="flex flex-row items-center justify-center mt-4">
-          <Pagination
-            totalCount={totalCount}
-            pageSize={PAGE_SIZE}
-            currentPage={currentPage}
-            onChange={onPageChange}
-            disabled={fetching}
-          />
+
+      <section className="bg-neutral-100">
+        <div className="max-w-screen-3xl 3xl:px-12 mx-auto flex flex-col py-0 xl:flex-row xl:items-center xl:gap-[139px]">
+          <div className="mx-auto px-6 py-24 sm:max-w-[533px] sm:px-0 lg:mx-0 lg:pl-24 xl:ml-32 xl:max-w-max xl:py-0 xl:pl-0">
+            <p>WELCOME</p>
+            <h1 className="my-3 text-5xl font-bold">
+              Time to make this site your own.
+            </h1>
+            <p>
+              Your new website is waiting to be built for something amazing.
+              <br />
+              What are you waiting for?
+            </p>
+            <div className="mt-8 flex flex-wrap gap-4 xl:flex-col 2xl:flex-row">
+              <Button size="large" className="w-fit">
+                Discover examples
+              </Button>
+              <Button size="large" className="w-fit" variant="secondary">
+                Read our Docs
+              </Button>
+            </div>
+          </div>
+          <div className="relative h-[490px] w-full sm:h-[640px] xl:max-w-[900px]">
+            <Image
+              src="/images/hero.png"
+              alt="Pantheon Logo"
+              fill
+              objectFit="cover"
+            />
+          </div>
         </div>
       </section>
+
+      <section className="max-w-screen-3xl mx-auto mt-32 flex justify-center px-4 sm:px-6 lg:px-0">
+        <HomepageArticleGrid articles={articles} />
+      </section>
     </Layout>
   );
 }
 
 export async function getServerSideProps() {
-  const {
-    data: articles,
-    totalCount,
-    cursor,
-  } = await PCCConvenienceFunctions.getPaginatedArticles({
-    pageSize: PAGE_SIZE,
-  });
+  const { data: articles } = await PCCConvenienceFunctions.getPaginatedArticles(
+    {
+      pageSize: 3,
+    },
+  );
 
   return {
     props: {
       articles,
-      cursor,
-      totalCount,
     },
   };
 }
diff --git a/starters/nextjs-starter-ts/pages/search.tsx b/starters/nextjs-starter-ts/pages/search.tsx
index d68eb9f..a161397 100644
--- a/starters/nextjs-starter-ts/pages/search.tsx
+++ b/starters/nextjs-starter-ts/pages/search.tsx
@@ -4,7 +4,7 @@ import {
 } from "@pantheon-systems/pcc-react-sdk";
 import { NextSeo } from "next-seo";
 import Image from "next/image";
-import { PostGrid } from "../components/grid";
+import ArticleGrid from "../components/grid";
 import Layout from "../components/layout";
 
 interface Props {
@@ -65,7 +65,7 @@ export default function Search({ articles, summary, searchString }: Props) {
         Search results for &quot;{searchString}&quot;
       </h3>
       <section>
-        <PostGrid contentType="posts" data={articles} />
+        <ArticleGrid articles={articles} />
       </section>
     </Layout>
   );
diff --git a/starters/nextjs-starter-ts/public/images/hero.png b/starters/nextjs-starter-ts/public/images/hero.png
new file mode 100644
index 0000000..5b766e1
Binary files /dev/null and b/starters/nextjs-starter-ts/public/images/hero.png differ
